pegasus/sites.v3/hourofcode.com/public/css/imports/_breakpoints.scss

```diff
@@ -0,0 +1,4 @@
+$width-lg: 1024px; // Laptop
```

Comment:
I have seen different cutoffs for these other devices. Do you know if we are eventually updating these everywhere?

Reply:
This is the standard breakpoint for a smaller laptop (13-15") — I'm not sure if we'll update it everywhere eventually, but I'm def open to it!

Reply:
Oh sorry I meant the tablet and mobile cutoffs! I'm thinking of application.scss where I did some work on the superhero banners this spring.

Reply:
Oh! I usually use these breakpoints since they align well w/ the Chrome inspector responsive viewer.